lpcompiler.pas

```diff
@@ -1702,7 +1702,10 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
         end;
 
         if (MethodOfObject(TLapeGlobalVar(OldDeclaration).VarType) <> MethodOfObject(Result.Method.VarType)) then
-          LapeException(lpeNoForwardMatch, Tokenizer.DocPos);
+        begin
+          if (not MethodOfObject(TLapeGlobalVar(OldDeclaration).VarType) and TLapeType_MethodOfObject(TLapeGlobalVar(OldDeclaration).VarType).HiddenSelf) then
```

Comment:
When does this happen?

Reply:
It's not needed, it must have been leftover code from before I handled it in other places.